src/matlab/create_nlp_nosnoc.m

```diff
@@ -140,13 +140,33 @@
 J_comp_fesd = 0;
 J_regularize_h = 0;
 J_regularize_sot = 0;
-% constraints
-g = {};
-lbg = [];
-ubg = [];
+
+% Initialize problem struct
+% TODO This may one day be a class!
+problem = struct();
+% Primal vars
+problem.w = {};
+problem.w0 = [];
+problem.lbw = [];
+problem.ubw = [];
+% Index vectors
+problem.ind_x = [];
+problem.ind_u = [];
+problem.ind_v = [];
+problem.ind_z = [];
+problem.ind_h = [];
+problem.ind_elastic = [];
+problem.ind_sot = []; % index for speed of time variable
+problem.ind_boundary = []; % index of bundary value lambda and mu
+problem.ind_t_final = []; % Time-optimal problems: define auxilairy variable for the final time.
+% Constraints
+problem.g = {};
+problem.lbg = [];
+problem.ubg = [];
+problem.ind_g_clock_state = [];
```

Comment:
I just checked out of curiosity, `ind_g_clock_state` seems to never be used.
Can be removed later of course.